lib/restHandlers.js

```diff
@@ -80,149 +80,206 @@
 	};
 
 	// GET /items
-	RestHandlers.prototype.listItems = function(object, params, callback) {
+	RestHandlers.prototype.listItems = function (object, params, callback) {
 		var self = this,
-			paramsToSupply = _.extend({}, _.isObject(params) ? params : {}, this.options.apiToken ? { api_token: this.options.apiToken } : {}),
-			dataObject = object == 'authorizations' ? { multipart: false, data: paramsToSupply } : { query: qs.stringify(paramsToSupply) },
-			req = rest[object == 'authorizations' ? 'post' : 'get'](apiUrl(object, this.options, false), dataObject);
-
-		req.on('complete', function(data, res) {
+			paramsToSupply = _.extend({}, _.isObject(params) ? params : {}, this.options.apiToken ? {api_token: this.options.apiToken} : {}),
```

Comment:
Please check the (auto)-formatting in all modified files. Looks like there's a lot of unnecessary changes, like removing spaces, for example `{api_token: this.options.apiToken}`

Reply:
Thanks, it was wrong formatting configuration in IDE indeed